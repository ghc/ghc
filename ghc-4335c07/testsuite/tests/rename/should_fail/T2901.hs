{-# LANGUAGE DisambiguateRecordFields #-}
-- Trac #2901

module T2901 where

f = F.Foo { F.field = "" }
