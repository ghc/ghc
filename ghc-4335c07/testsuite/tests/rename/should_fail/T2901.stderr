
T2901.hs:6:5: error:
    Not in scope: data constructor ‘F.Foo’
    No module named ‘F’ is imported.
